opta-mern-stack/docker-compose.yaml

```diff
@@ -0,0 +1,21 @@
+version: "3"
```

Comment:
do they need to use this file if they use `opta apply --local` there is no instructions what to do with it in our tutorial

Reply:
Removed docker-compose; doesnt make sense to confuse the user between local and docker-compose